datumaro/plugins/synthetic_data/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from datumaro.plugins.synthetic_data.image_generator import ImageGenerator
```

Comment:
```suggestion
```
In plugins, `__init__.py` can be empty.

Reply:
@AlexKoff88 suggested using short import like 
```python
from datumaro.plugins.synthetic_data import ImageGenerator
```
instead of
```python
from datumaro.plugins.synthetic_data.image_generator import ImageGenerator
```

Reply:
Ok, but keep in mind that plugins are not supposed to be called this way, in general.